packages/react-components/react-charts-preview/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -522,14 +519,7 @@ export const CartesianChart: React.FunctionComponent<IModifiedCartesianChartProp
       ref={(rootElem: HTMLDivElement) => (chartContainer.current = rootElem)}
       onMouseLeave={_onChartLeave}
     >
-      {!_isFirstRender.current && <div id={idForDefaultTabbableElement} />}
```

Comment:
why remove the !_isFirstRender condition

Reply:
This condition was used for the FocusZone which we are no longer using